src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3477,6 +3477,45 @@ void ApplicationManagerImpl::ProcessPostponedMessages(const uint32_t app_id) {
   std::for_each(messages.begin(), messages.end(), push_allowed_messages);
 }
 
+void ApplicationManagerImpl::ProcessOnDataStreamingNotification(
+    const protocol_handler::ServiceType service_type,
+    const uint32_t app_id,
+    const bool status) {
```

Comment:
@mkorniichuk status is unclear name. May be `send_required` or `send_needed` or something more meaningful?

Reply:
@AByzhynar fixed in 112037801c58a44cf62a8d078ab186fb9f25e7b5